Header header
float64 h
float64[] h_top_array
float64 h_clust
sensor_msgs/Image adjusted_image